lib/suse/toolkit/renderer.rb

```diff
@@ -7,11 +7,21 @@ def initialize(*arguments)
         super
       end
 
-      def render(filename, locals: binding)
-        bind = locals.is_a?(Binding) ? locals : OpenStruct.new(locals).instance_eval { binding }
-        @templates[filename] ||= ERB.new File.read(File.expand_path("../../connect/templates/#{filename}.erb", __FILE__)), 0, '-<>'
+      def render(filename, locals = {})
+        bind = binding
+        path = "../../connect/templates/#{filename}.erb"
+
+        locals.each_pair do |key, value|
+          bind.local_variable_set(key, value)
+        end
+
+        @templates[filename] ||= ERB.new File.read(File.expand_path(path, __FILE__)), 0, '-<>'
```

Comment:
Why this change?

Reply:
Yes this is my matter of taste. I basically only moved out the path as a own variable to make the line more short and more clear (at least for me). If you have objections I can also change it back.

Reply:
I see, actually my comment was more about the `, 0, '-<>'` at the end. What's that about?

Reply:
So after checking out the documentation: the zero is the safe_level which is actually saying that erb is rendering in a separate thread.
For the '-<>': It defines how erb should handle blank lines and spaces.
http://ruby-doc.org/stdlib-2.5.0/libdoc/erb/rdoc/ERB.html#method-c-new

Reply:
Ok, but I don't get it. We didn't have those before. Why do we need them now?

Reply:
I didn't see that they were already there before -- my mistake!